src/index.html

```diff
@@ -3,11 +3,362 @@
+                      placeholder="E-mail"
+                    >
+                    <button class="form__button">
+                      <img
+                        class="form__img"
+                        src="images/subscribe/arrow.png"
+                        alt="arrow"
+                      >
+                    </button>
+                  </label>
+                </form>
+              </div>
+            </div>
+          </div>
+        </section>
+      </div>
+    </main>
+
+    <footer class="footer" id="contucts">
+      <div class="container">
+        <div class="wrapper">
+          <div class="footer__content">
+            <div class="top-part">
+              <a href="#" class="top-part__logo">NA <br> MU</a>
+              <div class="top-part__icon">
+                <a
+                  href="https://www.facebook.com/sharer/sharer.php?u=https%3A//www.facebook.com/facebook/"
+                  class="top-part__icon--link"
+                  target="_blank"
+                ></a>
+                <a
+                  href="https://www.instagram.com/"
+                  class="top-part__icon--link top-part__icon--link-instagram"
+                  target="_blank"
+                ></a>
+              </div>
+            </div>
+
+            <div class="midlle-part">
+              <div class="midlle-part__right">
+                <p class="midlle-part__tittle">Графік роботи</p>
+                <ul class="midlle-part__schedual schedual">
+                  <li class="midlle-part__time">12:00 — 20:00 <span class="schedual__day">середа</span> </li>
+                  <li class="midlle-part__time">12:00 — 20:00 <span class="schedual__day">четвер</span> </li>
+                  <li class="midlle-part__time">12:00 — 20:00 <span class="schedual__day">п’ятниця</span></li>
+                  <li class="midlle-part__time">11:00 — 19:00 <span class="schedual__day">субота</span> </li>
+                  <li class="midlle-part__time">11:00 — 19:00 <span class="schedual__day">неділя</span> </li>
+                </ul>
+                <p class="midlle-part__subttitle">
+                  Вихідні понеділок, вівторок
+                  <br> Kaca припиняє роботу за 1 годину
+                  <br> до закриття музею
+                </p>
+              </div>
+              <div class="midlle-part__left">
+                <p class="midlle-part__tittle">Контакти:</p>
+                <div class="midlle-part__contacts">
+                  <a
+                    href="https://goo.gl/maps/4D3C4MMwGL1tGH4TA"
+                    class="midlle-part__adress"
+                    target="_blank"
+                  >
+                    вул. M. Грушевського, 6; Київ, 01001
+                  </a>
+                  <a href="tel:+38 (044) 278-13-57" class="midlle-part__adress">
+                      тел. +38 (044) 278-13-57
+                  </a>
+                  <a href="tel:+38 (044) 278-13-57" class="midlle-part__adress">
```

Comment:
Need to fix format phone everywhere only in href
```suggestion
                  <a href="tel:+380442781357" class="midlle-part__adress">
```